cmd/axtSam/axtSam.go

```diff
@@ -57,10 +57,10 @@ func axtToSam(axtfile string, header sam.Header, output string) {
 	defer writer.Close()
 
 	//setting up channels and wait groups
-	data, results := make(chan axt.Axt, 824), make(chan sam.Sam, 824)
```

Comment:
Are we dropping the channel creation here because we made the channel additions in the axt.go file?